deploy/package-operator/operator/kustomization.yaml

```diff
@@ -0,0 +1,12 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: observability-operator
+images:
+- name: observability-operator
+  newName: local-registry:30000/observability-operator/observability-operator
+  newTag: 0.0.16
```

Comment:
is this right? 🤔 
Also how is the version/tag updated ?

Reply:
Should also `make package` and `git diff --exit`  in our CI checks to ensure that packages are up to date?

Reply:
> is this right? thinking Also how is the version/tag updated ?

The tag is updated via `kustomize set image` iiuc.

> Should we also make package and git diff --exit in our CI checks to ensure that packages are up to date?

We don't actually store any package artifacts. The resources are generated with kustomize and an image is created from that. So no need to check if any generated files have changed.